media/Circletracer/index.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html lang="en">
+
+// MIT License
```

Comment:
`html` also accept these comment format?

Reply:
If there is a problem with the html format, I will change it to the <!-- --!>` format.